pkg/kubelet/cadvisor/cadvisor_linux.go

```diff
@@ -46,8 +47,17 @@ var _ Interface = new(cadvisorClient)
 // The amount of time for which to keep stats in memory.
 const statsCacheDuration = 2 * time.Minute
 const maxHousekeepingInterval = 15 * time.Second
+const defaultHousekeepingInterval = 10 * time.Second
 const allowDynamicHousekeeping = true
 
+func init() {
+	// Override the default cAdvisor housekeeping interval.
+	if f := flag.Lookup("housekeeping_interval"); f != nil {
```

Comment:
I guess I'm suggesting that we should have a kubelet flag instead of a cAdvisor flag for this and inject the value in the kubelet flag into cadvisor manager.